src/lxc/af_unix.c

```diff
@@ -56,7 +56,7 @@ int lxc_abstract_unix_open(const char *path, int type, int flags)
 	addr.sun_family = AF_UNIX;
 
 	len = strlen(&path[1]) + 1;
-	if (len >= sizeof(addr.sun_path) - 1) {
```

Comment:
Abstract unix sockets are `\0` terminated at the first byte and the last byte. So unless I'm missing something this code is correct and should stay the way it is.